cmd/notary/integration_test.go

```diff
@@ -838,21 +838,6 @@ func TestClientDelegationsPublishing(t *testing.T) {
 	_, err = runCommand(t, tempDir, "-s", server.URL, "publish", "gun")
 	require.NoError(t, err)
 
-	// Now remove this key, and make a new file to import the delegation's key from
-	require.NoError(t, os.Remove(filepath.Join(keyDir, canonicalKeyID+".key")))
-	tempPrivFile, err := ioutil.TempFile("/tmp", "privfile")
-	require.NoError(t, err)
-	defer os.Remove(tempPrivFile.Name())
-
-	// Write the private key to a file so we can import it
-	_, err = tempPrivFile.Write(privKeyBytesNoRole)
-	require.NoError(t, err)
-	tempPrivFile.Close()
-
-	// Import the private key, associating it with our delegation role
-	_, err = runCommand(t, tempDir, "key", "import", tempPrivFile.Name(), "--role", "targets/releases")
```

Comment:
Have a look at part 3 and see what you think. All the export/import stuff is path based now so I think the new tests cover this case in general, even if not specifically